engine/src/test/java/org/camunda/bpm/engine/test/api/history/BatchHistoricDecisionInstanceDeletionTest.java

```diff
@@ -626,6 +628,24 @@ public void shouldSetInvocationsPerBatchType() {
     configuration.setInvocationsPerBatchJobByBatchType(new HashMap<>());
   }
 
+  @Test
+  public void shouldSetExecutionStartTimeInBatchAndHistory() {
+    ClockUtil.setCurrentTime(TEST_DATE);
+
+    // when
+    Batch batch = historyService.deleteHistoricDecisionInstancesAsync(decisionInstanceIds, null);
+    helper.executeSeedJob(batch);
+    List<Job> executionJobs = helper.getExecutionJobs(batch);
+    // finish one job so both batch & history are available
+    helper.executeJob(executionJobs.get(0));
+
+    // then
+    HistoricBatch historicBatch = historyService.createHistoricBatchQuery().singleResult();
+    batch = rule.getManagementService().createBatchQuery().singleResult();
+    AssertUtil.assertEqualsSecondPrecision(batch.getExecutionStartTime(), TEST_DATE);
+    AssertUtil.assertEqualsSecondPrecision(historicBatch.getExecutionStartTime(), TEST_DATE);
```

Comment:
According to our [testing best practices](https://github.com/camunda/camunda-bpm-platform/wiki/Testing-Best-Practices-Java), we agreed on using AssertJ assertions.

Reply:
Fixed it.